src/index.js

```diff
@@ -122,6 +122,9 @@ var VERSION = process.env.npm_package_version;
  * @param {object} [overrides.fields] The Hosted Fields [`fields` options](http://braintree.github.io/braintree-web/{@pkg bt-web-version}/module-braintree-web_hosted-fields.html#~fieldOptions). Only `number`, `cvv`, `expirationDate` and `postalCode` can be configured. Each is a [Hosted Fields `field` object](http://braintree.github.io/braintree-web/{@pkg bt-web-version}/module-braintree-web_hosted-fields.html#~field). `selector` cannot be modified.
  * @param {object} [overrides.styles] The Hosted Fields [`styles` options](http://braintree.github.io/braintree-web/{@pkg bt-web-version}/module-braintree-web_hosted-fields.html#~styleOptions). These can be used to add custom styles to the Hosted Fields iframes. To style the rest of Drop-in, [review the documentation for customizing Drop-in](https://developers.braintreepayments.com/guides/drop-in/customization/javascript/v3#customize-your-ui).
  * @param {boolean} [clearFieldsAfterTokenization=true] When false, the card form will not clear the card data when the customer returns to the card view after a succesful tokenization.
+ * @param {object} [vault] Configuration for vaulting credit cards. Only applies  when using a [client token with a customer id](https://developers.braintreepayments.com/reference/request/client-token/generate/#customer_id).
+ * @param {boolean} [vault.showSaveCardToggle=false] When true, the card form will include an option to let the customer decide not to vault the credit card they enter.
+ * @param {boolean} [vault.defaultValueForVaulting=true] Whether or not to have the save card input pre-checked. When set to `false` with `showSaveCardToggle` set to `false`, then cards will not be vaulted upon tokenization.
```

Comment:
I'm not a big fan of this key name. I'd like the two properties to be pretty closely named to make it obvious. What about:
```
vault.saveCardToggleVisible=false
vault.saveCardToggleDefaultValue=true
```
or
```
vault.saveCardInputVisible
vault.saveCardInputDefaultValue
```

Reply:
The one trouble with this is that the default value option applies _even_ if the card input is not visible. This allows merchants to choose to display saved payment methods, but not vault cards automatically on the client.

If we change the name to more closely align, with that, we should remove that behavior. Are we :+1: with that?

Reply:
I think we should include the option to automatically vault a card on the client. This will help merchants with more complex integrations (verifications and Advanced Fraud Tools)

Reply:
Sorry, by "remove that behavior", I meant, we shouldn't respect this value setting if the option to _show_ the toggle is not enabled.

Reply:
Blade and I IRL'd. We think we should keep the current names since the behaviors are related but independent of each other. @quinnjn let us know if that works for you?

Reply:
Yep that works for me! Resolving